packages/rum-core/src/common/utils.js

```diff
@@ -398,6 +398,27 @@ function getPageLoadMarks() {
   }
 }
 
+function getDuration(start, end) {
+  if (isUndefined(end) || isUndefined(start)) {
+    return null
+  }
+  return parseFloat(end - start)
+}
+
+/**
+ * Get number of microseconds elapsed since January 1, 1970 00:00:00 UTC
+ *
+ * On browsers that support DOM High resolution timestamp, this function takes care of
+ * accounting for clock drift
+ */
+function microtime() {
+  let time = Date.now()
+  if (typeof performance.now === 'function') {
```

Comment:
we still need this since we are planning to test this feature via extension. comes handy with 7.3 APM server version. Lets remove it once we enable this feature by default on 7.5